Editor/TreasuredMapEditor.cs

```diff
@@ -311,46 +326,117 @@ private void InitializeGuidedTour()
-                        {
-                            if(EditorUtility.DisplayDialog("Error", "Custom export folder does not exist.", "Config", "Cancel"))
-                            {
-                                SettingsService.OpenUserPreferences("Preferences/Treasured SDK");
-                            }
+                    }
+                    catch (Exception e)
+                    {
+                        UnityEngine.Debug.LogError(e.Message);
+                    }
+
+                    EditorUtility.DisplayProgressBar("Installing Treasured CLI", "Installing the Treasured CLI from npm", 1f);
+                    EditorUtility.ClearProgressBar();
+
+                    // Get version of Treasured CLI
+                    string version = "";
+                    try
+                    {
+                        using (Process process = new Process()) {
+                            process.StartInfo.FileName = "treasured";
+                            process.StartInfo.Arguments = "--version";
+                            process.StartInfo.UseShellExecute = false;
+                            process.StartInfo.RedirectStandardOutput = true;
+                            
+                            process.Start();
+                            
+                            process.WaitForExit();
+                            version = process.StandardOutput.ReadToEnd();
                         }
-                    });
-                    menu.ShowAsContext();
+                    }
+                    catch (Exception e)
+                    {
+                        UnityEngine.Debug.LogError(e.Message);
+                    }
+
+                    // Show message
+                    if (!string.IsNullOrEmpty(version))
+                    {
+                        EditorUtility.DisplayDialog("Treasured CLI installed", "Treasured CLI version " + version + " installed successfully", "OK");
+                    }
+                    else
+                    {
+                        EditorUtility.DisplayDialog("Treasured CLI installation failed", "Treasured CLI installation failed. Please try again.", "OK");
+                    }
+                }
+                GUILayout.Space(10f);
+                Color oldColor = GUI.backgroundColor;
+                GUI.backgroundColor = new Color(0.3f, 1.0f, 0.6f);
+                if (GUILayout.Button(EditorGUIUtility.TrTextContentWithIcon("Export", $"Export scene to {TreasuredSDKPreferences.Instance.customExportFolder}/{_map.exportSettings.folderName}", "SceneLoadIn"), Styles.exportButton, GUILayout.MaxWidth(150)))
+                {
+                    try
+                    {
+                        Exporter.Export(_map);
+                    }
+                    catch (ValidationException e)
+                    {
+                        MapExporterWindow.Show(_map, e);
+                    }
                 }
+                GUI.backgroundColor = oldColor;
```

Comment:
It might make sense to refactor this color changing code into a IDisposable and use `using()` to manage it, similar to how `EditorGUI.DisabledGroupScope` works